### DISCUSSION TITLE
Warning Message: "Occupied! Resending in cb!"

### DISCUSSION THREAD

**Discussion by henjoe** [nanomq/nanomq#1120] [General]
Hi, 

I am having this kind of warning message, but I don't know what this means. 
![image](https://user-images.githubusercontent.com/49782655/228095783-0adf0290-0d74-4a5c-888c-94bf57818a60.png)

Is it a serious message?

Thanks!

Regards,
Henjoe

**Comment by wanghaEMQ** [comment]
It is NOT a serious warning. It just means that NanoMQ is sending the previous msg. So NanoMQ buffers the current msg and sends it when the previous msg is been sent.
